WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -138,3 +143,17 @@ inline static BOOL isSnapshotTypeAmongstGivenTypes(XCElementSnapshot* snapshot,
   }
   return NO;
 }
+
+inline static BOOL doesSnapshotHasMoreThanOneVisibleChildrenSnapshots(XCElementSnapshot* snapshot)
+{
+  NSArray<XCElementSnapshot *> *cellSnapshots = [snapshot fb_descendantsMatchingType:XCUIElementTypeCell];
```

Comment:
That will not work with simple scrollviews and (table/collection)views with accessibility stripped cells.